events/app/views/pages/event.njk

```diff
@@ -16,65 +16,94 @@
+        <span class="caps {{ {s: 1} | spacingClasses({padding: ['right']}) }}">Free</span>
+        <span class="border-left-width-1 border-color-pumice {{ {s: 1} | spacingClasses({padding: ['right', 'left']}) }}">
+          Enquire to book
+        </span>
+      </div>
 
-    {% if (formattedDateRange.size === 1) %}
-      <time>{{ dateRange.start | formatAndDedupeOnTime(dateRange.end) | joinDateStrings }}</time>
-    {% endif %}
-  {% endfor %}
+      <div class="{{ {s:'HNL4'} | fontClasses }}">
+        {% for dateRange in event.times %}
+          <div class="flex border-top-width-1 border-color-pumice {{ {s: 2} | spacingClasses({padding: ['top', 'bottom']}) }}">
+            {% set formattedDateRange = dateRange.startDateTime | formatAndDedupeOnDate(dateRange.endDateTime) %}
+            <span class="{{ {s: 2} | spacingClasses({margin: ['right']}) }}">{% icon 'other/calendar' %}</span>
+            <div>
+              <time class="block">
+                <b class="{{ {s:'HNM4'} | fontClasses }}">{{ formattedDateRange | joinDateStrings }}</b>
+              </time>
+              {% if (formattedDateRange.size === 1) %}
+                <time class="block">
+                  {{ dateRange.startDateTime | formatAndDedupeOnTime(dateRange.endDateTime) | joinDateStrings }}
+                </time>
+              {% endif %}
+            </div>
+          </div>
+        {% endfor %}
 
-  {# Intro #}
-  {% component 'standfirst', {body: event.intro | safe} %}
 
-  {# Description #}
-  {{ event.description | safe }}
+        {% set eventbriteIdentifier = event.identifiers | findObjInArray(key='identifierSchema', val='eventbrite-id') %}
+        {% if eventbriteIdentifier %}
+          <div class="border-top-width-1 border-color-pumice {{ {s: 2} | spacingClasses({padding: ['top', 'bottom']}) }}">
+            <a class="flex-inline flex--v-center btn {{ {s:'HNM4'} | fontClasses }} {{ {s: 4} | spacingClasses({padding: ['left', 'right']}) }}" href="https://www.eventbrite.com/e/{{ eventbriteIdentifier.value }}/">
+              <span>{% icon 'other/ticket' %}</span>
+              Book free tickets
+            </a>
+          </div>
+        {% endif %}
+      </div>
 
-  {# Booking enquiries #}
-  {% if event.bookingEnquiryTeam %}
-    {% set team = event.bookingEnquiryTeam %}
-    <a href="mailto:{{team.email}}?subject=Booking for {{ event.title }}">{% icon 'other/ticket', null, ['icon--teal'] %}Email to book</a>
-    <p>Email <a href="mailto:{{ team.email }}?subject=Booking for {{ event.title }}">{{ team.email }}</a> or call {{ team.phone }}</p>
+      <div class="border-top-width-1 border-color-pumice {{ {s: 2} | spacingClasses({padding: ['top', 'bottom']}) }}">
+        {% if eventbriteIdentifier %}
+        <h3 class="{{ {s:'HNM4'} | fontClasses }}">Booking information</h3>
+        <div class="{{ {s:'HNL4'} | fontClasses }}">
+          <p>
+            Please be aware that booking a ticket for a free event does not guarantee you place.
```

Comment:
@Heesoomoon judging from this info - this isn't true?
https://wellcomecollection.org/visit-us/events-tickets

It seems we do guarantee a ticket, but given what I saw, we turned down people who were late. Should that potentially be the messaging instead?

Reply:
Along with the waiting list policy as that applies to all ticketed events.

Reply:
Maybe something like:
> Reservations / tickets for our __free__ events could be given to people on our waiting list if you don't arrive on time to make sure that the maximum amount of people can benefit from our events with the least amount of disruption.

Which makes me wonder about "Tickets" for our free events, essentially we're actually running a reservation system.

Reply:
Ignore most of above - I just re-read the paragraph.

The thought around reservation vs ticketing stays though.